Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml

```diff
@@ -14220,6 +14220,15 @@
             <Image value="Assets\Textures\Wireframe-Protoss-Observer-Shield03.dds"/>
         </WireframeShield>
     </CActorUnit>
+    <CActorModel id="AP_Scophophobia" parent="ModelAnimationStyleContinuous">
+        <Inherits index="BaseModelScale" value="1"/>
+        <Inherits index="Scale" value="1"/>
+        <Inherits index="Visibility" value="1"/>
+        <On Terms="Behavior.AP_ScopophobiaTarget.On" Send="Create"/>
+        <On Terms="Behavior.AP_ScopophobiaTarget.Off" Send="AnimBracketStop BSD"/>
+        <Host Subject="_Unit"/>
+        <HostSiteOps Ops="SOpAttachCenter"/>
+    </CActorModel>

```

Comment:
Range actor for the ability?

Reply:
I don't think it's worth; the range is set to be the same as vision range, the observers will autocast it so it doesn't need conscious attention, the activation is obvious so strategy is straightforward, and the observers will often be selected to range actors will just add clutter.

I'm against but can be convinced otherwise.